src/images/patterns/bg-0.svg

```diff
@@ -0,0 +1 @@
+<svg width="1" height="1" viewBox="0 0 1 1" xmlns="http://www.w3.org/2000/svg"><title>bg-0</title></svg>
```

Comment:
Not sure if we want to potentially rename these to `bg-00.svg` since it seems like the ordering looks like:

1. bg-0.svg
1. bg-10.svg
1. bg-11.svg
1. ...
1. bg-19.svg
1. bg-2.svg
1. bg-20.svg

Currently as we add more themes the order will always slightly shuffle, where new themes are added in the middle.

Reply:
yeah good call